packages/IntegrationTest/tests/SampleTests.tsx

```diff
@@ -28,12 +28,6 @@ functionTest.skip('FailingExample', () => {
  * Test methods can also be async
  */
 functionTest('AsyncNativeModuleExample', async () => {
-  // Using AsyncStorage will YellowBox, which crashes on pre-1903 (including
```

Comment:
There's also one more instance of this in `MountComponentTests.tsx` for Picker. I can remove that after this PR though if you want.